### PR TITLE
Add dotnet 3.1 sidecar image

### PR DESCRIPTION
**Pull request by svor**
Signed-off-by: svor <vsvydenk@redhat.com>

Adds dotnet 3.1.301 sidecar image

### CODE REVIEW

File: .github/workflows/build-publish.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
```
Comment (ericwill) [inline .github/workflows/build-publish.yaml]: Copyright year.

File: .github/workflows/pr-check.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
```
Comment (ericwill) [inline .github/workflows/pr-check.yaml]: Copyright year.

File: etc/entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+#
+# Copyright (c) 2018-2018 Red Hat, Inc.
```
Comment (ericwill) [inline etc/entrypoint.sh]: Copyright year.

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,96 @@
+# Copyright (c) 2019 Red Hat, Inc.
```
Comment (ericwill) [inline Dockerfile]: Copyright year.

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+FROM node:10.16-stretch-slim
```
Comment (ericwill) [inline Dockerfile]: Would it be feasible to try and use the upstream [.NET Core image](https://hub.docker.com/_/microsoft-dotnet-core) as a base?

Comment (svor) [reply Dockerfile]: yep, tried it and it looks better! Even more the image based on upstream .NET Core is smaller on 87

Comment (ericwill) [reply Dockerfile]: Nice :+1:

**Review by ericwill — APPROVED** [review]

**Pull request merged**